### PR TITLE
Excel: Files: Fix issues with removing worksheets

### PR DESCRIPTION
**Pull request by osrjv**
- Fix IndexError when removing (certain) worksheets from XLS workbooks
- Reset active worksheet if it is deleted
- Ensure the last worksheet can not be deleted

Original issue reported internally by @TimothyLopas

### CODE REVIEW

File: packages/main/src/RPA/Excel/Files.py [hunk 1]
```diff
@@ -787,6 +787,14 @@ def append_worksheet(self, name=None, content=None, header=False, start=None):
 
     def remove_worksheet(self, name=None):
         name = self._get_sheetname(name)
+        others = [sheet for sheet in self.sheetnames if sheet != name]
+
+        if not others:
+            raise ValueError("Workbook must have at least one worksheet")
```
Comment (mikahanninen) [inline packages/main/src/RPA/Excel/Files.py:793]: Just curious - why there needs to be this kind of protection? I mean why user can't just delete all sheets and create then new sheet.

Comment (osrjv) [reply packages/main/src/RPA/Excel/Files.py:793]: At least .xls workbooks (with the upstream library) can not have zero worksheets, it breaks in spectacular ways. With .xlsx that can be done, as long as the user adds one before saving. Not sure if I want the keywords to behave differently based on the file extension.

Comment (mikahanninen) [reply packages/main/src/RPA/Excel/Files.py:793]: Yeah that is fine then to keep the protection. Thanks 👍

Comment (cmin764) [inline packages/main/src/RPA/Excel/Files.py:793]: Nit
```suggestion
            raise ValueError("Workbook must have at least one other worksheet")
```
if that makes more sense, so the user understands that his one and only worksheet isn't enough as it leaves the Excel empty.

**Review by cmin764 — APPROVED** [review]
LGTM!

Also, it is possible to link in this PR the reported issue? (if that's created already)

File: packages/main/src/RPA/Excel/Files.py (hunk 1 above)
Comment (osrjv) [reply packages/main/src/RPA/Excel/Files.py]: Fixed in both places

**Pull request merged**